controllers/exerciseController.js

```diff
@@ -56,7 +56,19 @@ const deleteExercises = async(req,res,next)=>{
   }
 }
 
+const getUserExercises = async(req,res,next)=>{
 
+  try {
+    const result = await Exercise.find({creator: req.userId})
+    res
+    .status(200)
+    .setHeader('Content-Type','application/json')
+    .json(result)
+  } catch (err) {
+    next(err)
```

Comment:
I would add some logging here in the catch block. If something went wrong, you would not know what happened.